src/main/resources/view/DarkTheme.css

```
@@ -49,6 +49,28 @@
     -fx-max-height: 0;
 }
 
+.tab-header-background {
+    -fx-background-color: derive(#1d1d1d, 20%);
+}
+
+.tab {
+    -fx-background-color: #1d1d1d;
+    -fx-background-radius: 10 10 0 0, 9 9 0 0, 8 8 0 0;
+    -fx-border-radius: 10 10 0 0, 9 9 0 0, 8 8 0 0;
```

```suggestion
    -fx-background-radius: 7 7 0 0;
    -fx-border-radius: 7 7 0 0;
```
i think the one i initially suggested a bit too round lol i suggest smooth a bit more, then remove the 2nd and 3rd param cos i dont really know what they do

---

Fixed in b4a4cc